client/asset/btc/spv_test.go

```diff
@@ -357,7 +357,7 @@ func (c *tNeutrinoClient) GetBlockHash(blockHeight int64) (*chainhash.Hash, erro
 func (c *tNeutrinoClient) BestBlock() (*headerfs.BlockStamp, error) {
 	c.blockchainMtx.RLock()
 	if c.getBestBlockHashErr != nil {
-		c.blockchainMtx.RUnlock()
+		defer c.blockchainMtx.RUnlock() // reading c.getBestBlockHashErr below
```

Comment:
Hit this too.  Had put a fix in a [draft PR for tangential work](https://github.com/decred/dcrdex/pull/1966) and forgotten about it: https://github.com/decred/dcrdex/pull/1966/commits/bf8a4e1fcb465abb3ab5532d8a4d74f80f701c94

We can go with your defer though.